server/controllers/UserAuth.js

```diff
@@ -24,5 +25,31 @@ class UserAuth {
           .json({ message: 'Registration Successful', token });
       }).catch(err => res.status(400).json(console.log(err)));
   }
+
+  /**
+     * This function register a new user
+     * @param {object} req - the request object.
+     * @param {object} res - The response object.
+     * @returns {object} Returns success message and jwt token.
+     */
+  loginUsers(req, res) {
+    authModel.userSignIn(req.body)
+      .then((result) => {
+        if (result.rowCount === 0) {
+          return res.status(404).json({ error: 'User does not exist!' });
```

Comment:
I don't think this is the right status code. Please confirm.

Reply:
ok. I would confirm and get back to you